lib/ZumoOta/src/App.h

```diff
@@ -111,6 +112,11 @@ class App
      * Instance of the Upload class 
      */
     Upload m_upload;
+
+    /**
+     * Instance of the NetworkCustom class 
+     */
+    NetworkCustom network;

```

Comment:
Missing m_ prefix. Please be consistent in naming rules.

Reply:
It will be performed by the next commit.